src/styles/main.scss

```diff
@@ -95,3 +95,11 @@ h1, h2, h3, h4, .navbar-nav {
     font-family: 'Ubuntu', serif;
   }
 }
+
+.footer-icon {
+  font-size: calc(1vw + 1vh);
+}
+
+.footer-line {
+  border-top: 1px solid #6c757d;
```

Comment:
I think you can use `$gray-600` here. See the docs on available color variables at https://getbootstrap.com/docs/4.5/getting-started/theming/#color